3rdparty/haskell/BUILD.ghc-lib-parser

```diff
@@ -52,7 +52,7 @@ haskell_library(
     "-I/compiler", "-I/compiler/utils"
   ],
   package_name = "ghc-lib-parser",
-  version = "8.8.1.20190916",
+  version = "8.8.1.20190905",
```

Comment:
This looks like a downgrade. :thinking:

Reply:
Yeah, I haven't created the new ghc-libs yet. I just wanted to see the failure with the one I created before.

Reply:
Of course, it doesn't get to running the tests because some DAML doesn't compile.